cmd/config-translator/translator.go

```diff
@@ -90,16 +89,16 @@ func main() {
 
 	mergedJsonConfigMap, err := cmdutil.GenerateMergedJsonConfigMap(ctx)
 	if err != nil {
-		panic(fmt.Sprintf("E! Failed to generate merged json config: %v", err))
+		log.Panicf("E! Failed to generate merged json config: %v", err)
 	}
 
 	if !ctx.RunInContainer() {
 		// run as user only applies to non container situation.
-		current, e := user.Current()
-		if e == nil && current.Name == "root" {
+		current, err := user.Current()
+		if err == nil && current.Name == "root" {
 			runAsUser, err := cmdutil.DetectRunAsUser(mergedJsonConfigMap)
 			if err != nil {
-				panic("E! Failed to detectRunAsUser\n")
+				log.Panicf("E! Failed to detectRunAsUser")
```

Comment:
If we aren't passing in any other parameters for the format string, this could just be:
```suggestion
				log.Panic("E! Failed to detectRunAsUser")
```

Reply:
Have not notice that. I used `log.Panic` for some but not noticing this until now. Nice catch